index.html

```diff
@@ -7,5 +7,15 @@
     <link rel="stylesheet" href="styles.css" />
     <script type="module" src="script.js"></script>
   </head>
-  <body></body>
+  <body>
+    <footer>
+      <div class="footer-links">
+        <a href="#">Account</a>

```

Comment:
Assuming these hrefs are still being defined

Reply:
Yes, please read the following issue/task/feature that this pull request seeks to resolve #8